tools/ci_build/github/azure-pipelines/linux-cpu-aten-pipeline.yml

```diff
@@ -33,7 +33,11 @@ jobs:
       Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_aten_cpu
       Context: tools/ci_build/github/linux/docker
       DockerBuildArgs: "--build-arg BUILD_UID=$( id -u )"
-      Repository: onnxruntimecpubuildaten
+      Repository: 'onnxruntimecpubuildaten'
+      UseImageCacheContainerRegistry: false
+      UsePipelineCache: true
+      DockerCacheKeys: 'tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_aten_cpu, tools/ci_build/github/linux/docker/manylinux*,
```

Comment:
Multiple paths need to be joined by "|", not ","

Reply:
"|" is for segments.